cowbird/monitoring/monitoring.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, config=None):
                        must be initialized with a proper config and after that the init function should not be hit.
         """
         if not config:
-            raise Exception("Without proper application settings, the Monitoring class cannot obtains a proper database"
-                            " store.")
+            raise ValueError("Without proper application settings, the Monitoring class cannot obtains a proper "
```

Comment:
I don't think I would put the `HandlerConfigurationException` here since the `monitoring` class is not part of the handlers. 
I could always create a custom `MonitoringConfigException` for this case, what do you think?

Reply:
Yes. Anything more specific than the base exceptions will help trace the source of the error.